src/scripts.js

```diff
@@ -65,9 +66,13 @@ return recipeCard;
 // //Example of one way to import functions from the domUpdates file. You will delete these examples.
 // import {exampleFunction1, exampleFunction2} from './domUpdates.js'
 window.addEventListener("load", () => {
-  dataLoad();
+  loadData();
 });
 
+window.addEventListener('click', (event) => {
+  updateUserRecipes(event)
+})
```

Comment:
any reason you chose `window` for this listener over `allRecipes`?

Reply:
Oversight on my part. Change made now.